lib/pos_mempool.go

```diff
@@ -1095,6 +1104,13 @@ func (mp *PosMempool) UpdateLatestBlock(blockView *UtxoView, blockHeight uint64)
 	}
 
 	mp.readOnlyLatestBlockView = blockView
```

Comment:
Should mp.readOnlyLatestBlockView also be a copy here? Now I'm questioning whether it's correct to use the input blockView pointer as-is